pubspec.yaml

```diff
@@ -4,7 +4,7 @@ author: "Dart Team <misc@dartlang.org>"
 homepage: https://github.com/dart-lang/http
```

Comment:
This should be 0.12.0-dev. We use the "-dev" suffix to clearly distinguish between released versions and those that are still in progress.